x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/index.tsx

```diff
@@ -137,56 +139,73 @@ export const ManagementList = () => {
       },
       {
         field: '',
-        name: i18n.translate('xpack.endpoint.management.list.lastActive', {
+        name: i18n.translate('xpack.endpoint.host.list.lastActive', {
           defaultMessage: 'Last Active',
         }),
+        dataType: 'date',
         render: () => {
           return 'xxxx';
         },
       },
     ];
   }, [queryParams, history]);
 
+  const HostPage = styled(EuiPage)`
+    padding: 0;
+  `;
+
+  const HostHeader = styled(EuiPageHeader)`
```

Comment:
Same here and below - move the `styled` component definition to be module level.